framework/src/MonteCarloPlayer.cpp

```diff
@@ -0,0 +1,139 @@
+#include "MonteCarloPlayer.h"
+#include "GameTypes.h"
+#include "RandomPlayer.h"
+
+namespace Player {
+
+// Select a move from the given boardstate
+Game::move_t MonteCarloPlayer::selectMove(Game::GameBoard& board, playernum_t playerNum)
+{
+    m_rootNode = std::shared_ptr<MonteCarlo::TreeNode>(new MonteCarlo::TreeNode());
+    m_rootNode->boardState = board;
+    m_rootNode->playerNum = playerNum;
+    m_rootNode->simulated = true; // no use in simulating this node
+    m_selectedNode = m_rootNode;
+
+    runSearch(1000); // TODO: udpate way for number of iterations to be set
```

Comment:
This doesn't need to be defined by the user, so a define statement or an enum declaration in the MonteCarloPlayer or MonteCarloTypes header files should be sufficient. Going ahead and moving it there  would probably be good.

Reply:
yep. added it to the MonteCarloTypes header in the latest commit.